saml-core-api/src/main/java/org/keycloak/dom/saml/v2/metadata/ExtensionsType.java

```diff
@@ -88,4 +89,14 @@ public void removeExtension(Object extension) {
     public List<Object> getAny() {
         return Collections.unmodifiableList(this.any);
     }
-}
\ No newline at end of file
+
+    public EntityAttributes getEntityAttributes() {
+        for (Object o : this.any) {
+            if (o instanceof EntityAttributes) {
+                return (EntityAttributes) o;
+            }
+        }
+        return null;
```

Comment:
```suggestion
        return this.any.stream()
            .filter(EntityAttributes::isInstance)
            .findAny();
```

Reply:
I agree. However, for adding this I must change maven.compiler.source of this subproject. I believe this is the best solution. Do you want to do it?
Otherwhise a solution could be:

return (EntityAttributes) this.any.stream().filter(o -> o instanceof EntityAttributes).findAny().orElse(null);

Reply:
Ah, ignore it for this moment then, I didn't realize this is still used by JDK 7 code.